src/Console/Command/FixCommand.php

```diff
@@ -368,31 +373,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ->setTime($fixEvent->getDuration())
         ;
 
-        $reporter = ReporterFactory::create()
-            ->registerBuiltInReporters()
-            ->getReporter($resolver->getFormat())
-        ;
-
         $output->write(
             $reporter->generate($reportSummary)
         );
 
+        if (null !== $stdErr) {
+            $stdErr = $output;
```

Comment:
let us keep it that way:
1. if we could use it stdErr we use it. it's the most common case.
2. if for some rare envs we can't use stdErr and we are using not-txt format we don't print it, as we have no place to do it. putting it inside JSON (/or whatever) is worse, as it would made whole output non-usable.